data/ecosystems/n/noir-lang.toml

```diff
@@ -143,3 +143,225 @@ url = "https://github.com/vezenovm/simple_shield"
 
 [[repo]]
 url = "https://github.com/whitenois3/nplate"
+
+[[repo]]
+url = "https://github.com/BattleZips/BattleZips-Noir"
+
+[[repo]]
+url = "https://github.com/colinnielsen/ecrecover-noir"
+
+[[repo]]
+url = "https://github.com/Ethan-000/halo2_backend"
+
+[[repo]]
+url = "https://github.com/Miyamura80/ZKMicrophone"
+
+[[repo]]
+url = "https://github.com/resurgencelabs/types"
+
+[[repo]]
+url = "https://github.com/SetProtocol/noir-rsa"
+
+[[repo]]
+url = "https://github.com/tonk-gg/dappicom"
+
+[[repo]]
+url = "https://www.noirguardians.io/"
```

Comment:
not a valid repo URL

Reply:
Thanks! Sorry for the non-github links, I'll fix that now 👍